README.md

```diff
@@ -197,6 +197,9 @@ export interface IUtils<TDate> {
   getSeconds(value: TDate): number;
   setSeconds(value: TDate, count: number): TDate;
 
+  getDate(value: TDate): number;
+  setDate(value: TDate, count: number): TDate;
```

Comment:
WDYT about making it `getDay`. I think it is more clear in terms of Date is the term for javascript date and may be confusing that it returns a number.

Reply:
Yes getDayInMonths or simply `getDay` would be fine. Everything else is great, so once you will done the rename – we can merge

Reply:
I based my naming on the javascript `Date` interface

`new Date().getDay()` => the day in the week
`new Date().getDate()` => the day in the month

But I don't have a strong opinion here
Libraries are not even coherent between them.

Reply:
@dmtrKovalenko